server/routes/appointments/create-and-edit/editAppointmentJourney.ts

```diff
@@ -2,6 +2,7 @@ import { PrisonerSummary } from '../../../@types/activitiesAPI/types'
 import { AppointmentCancellationReason, AppointmentApplyTo } from '../../../@types/appointments'
 
 export type EditAppointmentJourney = {
+  appointmentName: string
```

Comment:
Not that keen on this adding this to the session, but I don't think there's a good alternative.

Reply:
I think it's OK to have it in session but I wonder about moving it to the appointment journey. I could see it being used outside of edit - it could be populated during create and used in the caption for example.